pootle/apps/pootle_store/models.py

```diff
@@ -1861,13 +1867,6 @@ def update(self, overwrite=False, store=None, only_newer=False, user=None,
             self.parse(store=store)
             return
 
-        disk_mtime = self.get_file_mtime()
-        if only_newer and disk_mtime == self.file_mtime:
```

Comment:
Yes, but `update()` doesn't call `sync()` and `sync()` doesn't call `update()`. So unless there is any place where `update()` is being called with `only_newer`, `only_newer` should be safe to remove.